doc/Makefile.am

```diff
@@ -196,12 +196,12 @@ geany-sciwrappers-gtkdoc-tmp.h: geany-sciwrappers-gtkdoc.h
 		-e 's,sci_,scintilla_object__GI__MARK_,g' \
 		$< > $@
 
-GeanyScintilla-1.0.gir: GeanyScintilla-1.0-pre.gir
+GeanyScintilla-1.0.gir: GeanyScintilla-1.0-tmp.gir
 	$(AM_V_GEN) cat $< \
 		| $(SED) -e 's,scintilla_object__GI__MARK_,sci_,g' \
```

Comment:
it's a custom mark first added to *geany-sciwrappers-gtkdoc-tmp.h* so we can easily catch the methods we need to rename, without messing up with get_type or others like that.